packages/components/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Enhancements
 
 -   `InputControl`/`SelectControl`: update `height`/`min-height` to `32px` instead of `30px` to align with modern sizing scale ([#55490](https://github.com/WordPress/gutenberg/pull/55490)).
+-   `ToolsPanel`/`ToolsPanelHeader`: Added `dropdownMenuProps` to allow customization of the panel's dropdown menu ([#55785](https://github.com/WordPress/gutenberg/pull/55785)).
```

Comment:
We should probably also mention changes to how menu items are grouped in the dropdown menu?

Reply:
✅ https://github.com/WordPress/gutenberg/pull/55785/commits/3b001c24e93c3fc2750f036019c41d80e7c780b5